src/main/java/weavers/siltarae/mistake/service/MistakeService.java

```diff
@@ -46,7 +52,15 @@ public MistakeListResponse getMistakeList(Long memberId, MistakeListRequest requ
         List<Tag> tags = getTags(tagIds, memberId);
         Page<Mistake> mistakes = getMistakes(memberId, tagIds, request.toPageable());
 
-        return MistakeListResponse.from(mistakes, tags);
+        return MistakeListResponse.from(getMistakeResponseList(mistakes.getContent()), tags, mistakes.getTotalElements());
+    }
```

Comment:
```suggestion
        List<MistakeResponse> mistakeResponses = getMistakeResponseList(mistakes.getContent());
        return MistakeListResponse.from(mistakeResponses, tags, mistakes.getTotalElements());
    }
```

이쪽은 지금도 알아보기 쉽긴 하지만...! 코드의 일관성을 중요시한다면 이렇게 수정할 수 있을 것 같아요 🙌